### DISCUSSION TITLE
etcd is continuously restarting which booting from talos image

### DISCUSSION THREAD

**Discussion by ghost** [siderolabs/talos#4002] [Q&A] [answered]
etcd is continuously restarting which booting from talos image. Did anyone face this issue?

![image](https://user-images.githubusercontent.com/58803841/128014126-483aa2d7-a6e0-4023-97bb-51dea488a9c1.png)

**Comment by smira** [comment, marked as answer]
`talosctl logs etcd` should provide more information, but usually that means there's some member information inconsistency.

Depending on the cluster configuration (HA or not), there might be different ways to fix the membership.

**Reply by ghost** [reply]
Thank you for the response @smira .

I am not able to reach Talos, getting below error

root@hpe-sa:~# talosctl logs etcd
error fetching logs: rpc error: code = Unavailable desc = connection closed

**Reply by smira** [reply]
does it happen on the first boot? how are you booting Talos? what is the environment?

one guess is that the server disk is not wiped, so it's not new Talos installation, but rather Talos tries to boot from old data

**Reply by ghost** [reply]
Yes, you are correct. It's a reinstallation scenario. We are using HPE EL1000.

I am attaching the talos bin file as "Image File Removable Media" and setting the first boot as "Generic USB".

Please suggest.

**Reply by andrewrynhard** [reply]
@malayamithun . You will likely need to wipe the disk. If you have access to the API, `talosctl reset --system-labels-to-wipe=EPHEMERAL -n $IP`, or booting a live cd (another distro) and running `wipefs -a`.
